android/gradle.properties

```diff
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
+android.enableDexingArtifactTransform=false
```

Comment:
why do we need this? https://github.com/Lyokone/flutterlocation/issues/802

Reply:
While I was debugging I saw an issue related to flutter google maps (https://github.com/flutter/flutter/issues/72185) so I added this as an extra check.

Reply:
This looks very old, can you remove the line test and confirm if it works without it? Thanks :3

Reply:
Removed the line, works well without it as well. Thanks